js-jasmine/spec/gilded_rose_spec.js

```diff
@@ -1,11 +1,73 @@
 var Shop = require('../src/shop.js');
 var Item = require('../src/item');
+
 describe("Gilded Rose", function() {
 
   it("should foo", function() {
     const gildedRose = new Shop([ new Item("foo", 0, 0) ]);
-    const items = gildedRose.updateQuality();
+    const items = gildedRose.quality();
     expect(items[0].name).toEqual("foo");
   });
 
+  it('Quality should be positive', function(){
+    const gildedRose = new Shop([ new Item("foo", 0, 0) ]);
+    const items = gildedRose.quality();
+    expect(items[0].quality).toBeGreaterThanOrEqual(0);
+  })
+  
```

Comment:
One improvement could be to include a README explaining how to run your project etc.